jobs/germany-covid-forecast/germany-crowd-forecast.sh

```diff
@@ -0,0 +1,25 @@
+#!bin/bash
+
+# check repository is up to date and update dependencies
+/bin/bash schedule/jobs/germany-covid-forecasts/update-dependencies.sh
+
+# create crowd forecast app submission data
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript human-forecast/update-submissions.R
+
+# Redeploy forecast app (to update submission date to next week)
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript human-forecast/redeploy-forecast-app.R
+
+# update Rt crowd forecast samples
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript rt-crowd-forecast/update-samples.R
```

Comment:
rather than doing it this way can a bash update script be written in the `rt-crowd-forecast` rather than doing so many things here.